Libs/MRML/DisplayableManager/vtkMRMLThreeDViewInteractorStyle.cxx

```diff
@@ -110,7 +111,8 @@ bool vtkMRMLThreeDViewInteractorStyle::DelegateInteractionEventToDisplayableMana
   // Get display and world position
   int* displayPositionInt = this->GetInteractor()->GetEventPosition();
   vtkRenderer* pokedRenderer = this->GetInteractor()->FindPokedRenderer(displayPositionInt[0], displayPositionInt[1]);
```

Comment:
`FindPokedRenderer` is error-prone. If somebody adds any content to the render window in a new renderer then it may break picking and or interaction with other widgets. It would be safer to pass the main 3D content renderer, not whatever renderer happened to be found at that position.

Reply:
Agreed, this can be improved and I will create a dedicated issue to improve and address this. In prefer to keep the existing behavior and focus this pull request on only revisiting the observation of the interactor style events.